functions/index.js

```diff
@@ -3,14 +3,16 @@ const express = require('express');
 const cors = require('cors');
 
 const app = express()
+const authGuard = require('./util/authGuard')
 
 app.use(cors())
 
-const { loginUser, signUpUser } = require('./controllers/user/index')
+const { login, signUp, image, getAll } = require('./controllers/user/index')
 
 //user Routes
-app.post('/signup', signUpUser);
-app.post('/login', loginUser);
-
-exports.api = functions.https.onRequest(app);
+app.post('/signup', signUp);
+app.post('/login', login);
+app.post('/user/image', authGuard, image);
+app.get('/users', getAll)
```

Comment:
Can you extract this to a router/index.js

and call that class in your index,js

you want to make sure your index.js is as stable as possible and you are not making plenty changes here